<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Lehrer">
     <attribute name="title">
      <string>Lehrer</string>
     </attribute>
     <widget class="QLineEdit" name="NameL_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="NameL_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="NachnameL_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nachname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NachnameL_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="AlterL_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="AlterL_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Fach_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fach</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Fach_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="OutputL">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>256</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SpeichernL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
     <widget class="QLabel" name="LehrerID_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LehrerID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LehrerID_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="LoeschenL">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Löschen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadL">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>SuS</string>
     </attribute>
     <widget class="QLabel" name="AlterS_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NachnameS_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="LoeschenS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Löschen</string>
      </property>
     </widget>
     <widget class="QListWidget" name="OutputS">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>256</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="NameS_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Klasse_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="AlterS_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="SuSID_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SuSID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Klasse_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Klasse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SuSID_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SpeichernS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameS_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="NachnameS_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nachname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Raum_I">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Raum_L">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Raum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadS">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>LehrerID_I</tabstop>
  <tabstop>NameL_I</tabstop>
  <tabstop>NachnameL_I</tabstop>
  <tabstop>AlterL_I</tabstop>
  <tabstop>Fach_I</tabstop>
  <tabstop>SpeichernL</tabstop>
  <tabstop>LoeschenL</tabstop>
  <tabstop>OutputL</tabstop>
  <tabstop>LoadL</tabstop>
  <tabstop>SuSID_I</tabstop>
  <tabstop>NameS_I</tabstop>
  <tabstop>NachnameS_I</tabstop>
  <tabstop>AlterS_I</tabstop>
  <tabstop>Klasse_I</tabstop>
  <tabstop>Raum_I</tabstop>
  <tabstop>SpeichernS</tabstop>
  <tabstop>LoeschenS</tabstop>
  <tabstop>OutputS</tabstop>
  <tabstop>LoadS</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
